packages/desktop-client/src/components/HelpMenu.tsx

```
@@ -57,9 +57,11 @@ const HelpButton = forwardRef<HTMLButtonElement, HelpButtonProps>(
         ref={ref}
         onPress={onPress}
         style={{
+          color: theme.pageText,
```

Biome (2.4.16)

[error] This property is later overwritten by an object member with the same name.

(lint/suspicious/noDuplicateObjectKeys)